src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -3,6 +3,7 @@
 from concurrent.futures import ThreadPoolExecutor, as_completed
 from dataclasses import asdict
 from datetime import UTC, datetime, timedelta
+from sqlite3 import OperationalError
```

Comment:
This import looks wrong to me since we don't use sqlite3, though it might be unintentionally working due to some aliasing

Reply:
good catch, thanks! I've changed it to import from `django.db.utils import OperationalError`